app/state_machines/workflows/conditionals.rb

```diff
@@ -11,4 +11,25 @@ def remove_response
       'awaiting_dispatch'
     end
   end
+
+  def unflag_for_clearance
+    if @user.disclosure_specialist?
+      'standard'
+    else
+      if @kase.assignments.where(team_id: BusinessUnit.dacu_disclosure.id).any?
+        'trigger'
+        else
+        'standard'
+      end
+    end
+  end
+
+  def transition_unflag_for_clearance
```

Comment:
How about a name that's more telling of it's function? e.g. `unflag_for_clearance_next_state`